docs/finops-hub/README.md

```diff
@@ -40,24 +42,22 @@ To learn more, see [FinOps hub template details](template.md).
 
 <br>
 
-## ➕ Create a new hub
+## 🧰 Create a new hub
 
 1. [Deploy the **finops-hub** template](../deploy).
-2. [Create a new cost export](https://learn.microsoft.com/azure/cost-management-billing/costs/tutorial-export-acm-data?tabs=azure-portal) using the following settings:
-   - **Metric** = `Amortized cost`
-   - **Export type** = `Daily export of month-to-date costs`
-     > 💡 _**Tip:** Configuring a daily export starts in the current month. If you want to backfill historical data, create a one-time export and set the start/end dates to the desired date range._
-   - **Storage account** = (Use subscription/resource from step 1)
-   - **Container** = `msexports`
-   - **Directory** = (Use the resource ID of the scope you're exporting without the first "/")
-3. Run your export.
-   - Exports can take up to a day to show up after first created.
-   - Use the **Run now** command at the top of the Cost Management Exports page.
-   - Your data should be available within 15 minutes or so, depending on how big your account is.
-4. Download one or more of the available Power BI starter templates:
+
+## 🛠️ Configure  exports
```

Comment:
Can we go back to a simple bulleted list instead of a separate section? 
```suggestion
2. [Configure scopes to pull cost data from](./configure-scopes.md).
3. Download one or more of the available Power BI starter templates:
```